less/components/illustration.less

```diff
@@ -1,15 +1,28 @@
 .illustration {
+  overflow: auto;
+  padding: 0 0 2rem 0;
   &.vertical-layout {
     flex-direction: column;
     .image-container {
       padding-right: 0;
     }
   }
-  .img-circle {
```

Comment:
O blug. Maybe we should call it `illustration-img-circle` instead of `circular-image` so that it doesn't become too confusing? And mention in a comment above the rule definition that we're calling it that so we don't collide w/ Bootstrap's namespace?